tests/feinterface/fe_interface_values_13.cc

```diff
@@ -0,0 +1,95 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2023 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+// Set up a two cells with different elements, such that neither element
+// dominates. Create an FEInterfaceValues object and call reinit on it.
+
+#include <deal.II/base/quadrature_lib.h>
+
+#include <deal.II/fe/fe_interface_values.h>
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_system.h>
+
+#include <deal.II/grid/tria.h>
+
+#include "../tests.h"
+
+#include "../test_grids.h"
+
+
+template <int dim>
+void
+test()
+{
+  Triangulation<dim> tria;
+  TestGrids::hyper_line<dim>(tria, 2);
+
+  const FESystem<dim> fe_left(FE_Q<dim>(1), FE_Q<dim>(2));
+  const FESystem<dim> fe_right(FE_Q<dim>(2), FE_Q<dim>(1));
+
+  hp::FECollection<dim> fe_collection;
+  fe_collection.push_back(fe_left);
+  fe_collection.push_back(fe_right);
+
+  const QGauss<dim - 1>          quadrature(3);
+  const hp::QCollection<dim - 1> q_collection(quadrature, quadrature);
```

Comment:
I think I know how to deal with this case because even though neither element dominates the other, it doesn't matter: The associated quadrature objects are the same: we should just use either one! Would you mind using different quadrature formula objects?

Reply:
Sure! And to justify the change, let's also increase the poly degree of one of the elements. They still won't dominate each other.